test/support/repo.ex

```diff
@@ -6,9 +6,14 @@ defmodule Authsense.Test.Repo do
   end
   def get(_model, _id), do: nil
 
-  def get_by(_model, email: "nobody@gmail.com"), do: nil
-  def get_by(_model, email: "rico@gmail.com") do
-    valid_resource "rico@gmail.com", "foobar"
+  def get_by(model, email: email) when is_atom(model), do: Enum.find(all(), &(&1.email == email))
```

Comment:
@rstacruz this is the one. I can't find a way to remove this yet.

I have attempted

```ex
def get_by(query = %Ecto.Query{}, email: email) do
# I can't get the query params here
end
```